crosswalks/Citation_File_Format_1.2.0.README.md

```diff
@@ -0,0 +1,243 @@
+# Crosswalk for Citation File Format 1.2.0
+
+## Format information
+
+Citation File Format (CFF) is a YAML format for providing citation metdaata for software (and experimentally for datasets).
+
+- Website: <https://citation-file-format.github.io/>
+- Specifications: <https://github.com/citation-file-format/citation-file-format/blob/1.2.0/schema-guide.md>
+- Schema: <https://github.com/citation-file-format/citation-file-format/blob/1.2.0/schema.json> (JSON Schema)
+
+## Crosswalk specifics
+
+The following sections provide details for directional mapping 
+in cases where mapping is not bidirectionally unambiguous.
+
+### `downloadUrl` <=> `repository-artifact` (and potentially others)
+
+- CodeMeta `downloadUrl`: "If the file can be downloaded, URL to download the binary."
+- CFF `repository-artifact`: "The URL of the work in a build artifact/binary repository."
+
+While the scope of `repository-artifact` is documented to be somewhat narrower than `downloadUrl`, 
+its main use case is discovery (e.g. for reuse and reproducibility),
+which is arguably the same as for `downloadUrl`.
+Therefore the two can be assumed to signify the same.
+
```

Comment:
downloadUrl should be a direct link to download, 
the potentially others links here should be used in codemeta's "url" instead of "downloadUrl"

Reply:
I see, so direct links only. I'll adapt accordingly.
Can there be > 1 `downloadUrl` in any given codemeta.json file?

Reply:
That is, I'll change the README accordingly, as `url`s in CFF can potentially be directo download links.

Reply:
Fixed in crosswalk and README in 89b2676e61c0ad4ac2518d98611c8c70bea2f122.